web/src/components/RelatedProducts/RelatedProducts.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC } from "react";
+
+import { GridItem } from "@nice-digital/nds-grid"; // Assuming GridItem is from this library
+import { Panel } from "@nice-digital/nds-panel";
+
+import { RelatedProductList } from "@/feeds/publications/types";
+
+import styles from "./RelatedProducts.module.scss";
+
+export type RelatedProductsProps = {
+	relatedProducts?: RelatedProductList[];
+};
+
+export const RelatedProducts: FC<RelatedProductsProps> = ({
+	relatedProducts,
+}) => {
+	return relatedProducts && relatedProducts.length > 0 ? (
+		<GridItem cols={12} md={4}>
```

Comment:
Could we move the grid component to the index page instead of inside the relatedProducts component?